CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ The submitted code should adhere to the following:
 ### Do's
 
 * Keep pull requests focused on addressing one issue or feature. If multiple separate issues
-  are fixed by a pull request, prefer commiting the fixes under multiple commits, one for each
+  are fixed by a pull request, prefer committing the fixes under multiple commits, one for each
```

Comment:
another pr for this too?

Reply:
Discussed offline. We are going to put these minor doc change in the same PR to reduce extra migration work from internal to external.